admin/src/app/shared/services/barcodescan.service.ts

```diff
@@ -0,0 +1,92 @@
+import { Injectable } from "@angular/core";
+import { APIQueueDatabase } from "./indexdb.service";
+import Dexie from "dexie";
+import { OrgModelApi } from "../lb-sdk";
+import { ToastrService } from "ngx-toastr";
+
+export interface BarcodeReceive {
+  sku: string;
+  done: boolean;
```

Comment:
I dont see any use of `done` argument  in this file ?

Why we have less parameters in `skus` table but the type has more parameters ? 
Please add the missing parameters in skus table definition in db class

Reply:
We don't need any more data in skus. It works expected now